src/content/en/2021/http.md

```diff
@@ -8,6 +8,7 @@ analysts: [tunetheweb]
 editors: []
 translators: []
 results: https://docs.google.com/spreadsheets/d/1pCdpndXTXexSIZLmVc-aUbp5PcHhZ83hbEEfmDyfD0U/
+dominiclovell_bio:
```

Comment:
@dominiclovell could you write a sentence or two about yourself here?

This will show at the bottom of the chapter. Example from last year to give you inspiration: https://almanac.httparchive.org/en/2020/http#authors